### ISSUE TITLE
Completes without output of masked sequences

### ISSUE THREAD

**Issue by zrlewis** [nextgenusfs/redmask#1]
Hi,

Thanks for writing this helpful wrapper. It was very easy to run. 

My issue is that the fasta file of masked sequences wasn't written. My output files are:

```
*.fa.repeats.bed
*.softmasked.fa
```

Here is the stdout:
```
cat RED.27125724.out
[Dec 06 09:55 AM] Running Python v2.7.13 
[Dec 06 09:56 AM] Loading assembly with N50 of 80,913 bp
[Dec 06 09:56 AM] Splitting genome assembly into training set (contigs > 1000 bp)
[Dec 06 09:59 AM] Finding repeats with Red (REpeat Detector)
[Dec 06 11:54 AM] Collecting results from Red
[Dec 06 11:54 AM] Summarizing results and converting to BED format

Masked genome: scaffolds_RED.fa.softmasked.fa
num scaffolds: 54,313
assembly size: 2,854,780,185 bp
masked repeats: 2,201,955,364 bp (77.13%)
```

It looks like the `--msk` flag is present in the command. Have you seen this behavior? Do you have any suggestions?

Thanks,
Zack

**Comment by nextgenusfs** [comment]
Hi Zack,
So which files were written?  Should be 3 output files - the repeats bed file, novel repeats fasta file, and then softmasked fasta file.  The will all have the "basename" of whatever you sepecified to the `-o` option, so I'm guessing your command looked something like this?
```
redmask.py -i input.fasta -o scaffolds_RED.fa
```
This should output the following then:
```
scaffolds_RED.fa.softmasked.fa
scaffolds_RED.fa.repeats.bed
scaffolds_RED.fa.repeats.fasta
```
I think the softmasked genome must be present, as that is what it used to calculate those masking stats.  Side note -- RED seems to be very "greedy" about finding repeats, you may need to play around with the parameters to get something reasonable.

Jon

**Comment by zrlewis** [comment]
Hi @nextgenusfs, 

Thanks for your response. Yes, that was my command. 

I was missing the `scaffolds_RED.fa.repeats.fasta` file. The other two were present.

Cheers,
Zack

**Comment by nextgenusfs** [comment]
Hi Zack, please try latest commit -- should be fixed ac36368bf7073dbc5df59329a8563de8ec7b499b

**Comment by zrlewis** [comment]
@nextgenusfs 

That works. Thanks!
